### DISCUSSION TITLE
[blocks]: Stepper

### DISCUSSION THREAD

**Discussion by TeodorPetrovic** [shadcn-ui/ui#3498] [Blocks Request]
### Description

Could you make a block like this stepper.
![image](https://github.com/shadcn-ui/ui/assets/63924993/65bd1867-08e6-4a3d-adb6-cea766e3ba5c)

**Comment by alexabstreiter** [comment]
There is one here that is already built out very well: https://github.com/shadcn-ui/ui/pull/318
